typings/index.d.ts

```diff
@@ -996,6 +996,7 @@ declare module 'klasa' {
 		public promptTime: number;
 		public promptLimit: number;
 		public quotedStringSupport: boolean;
+		public responses: Collection;
```

Comment:
reasonably sure this needs <string, KlasaMessage>, but I defer to @kyranet

Reply:
Exactly, since `Collection<K, V> extends Map<K, V>`, it's a generic, and must have both values specified. In this case it'd be:

```ts
public responses: Collection<string, KlasaMessage>;
```

Reply:
done